model/src/pyrenew/latent/infection_functions.py

```diff
@@ -124,16 +124,38 @@ def sample_infections_with_feedback(
     Returns
     -------
     tuple
-        A tuple `(Rt_adjusted, infections)`,
+        A tuple `(infections, Rt_adjusted)`,
         where `Rt_adjusted` is the infection-feedback-adjusted
         timeseries of the reproduction number R(t) and
         infections is the incident infection timeseries.
+
+    Notes
+    -----
+    This function implements the following renewal process:
+
+    .. math::
+
+        I(t) & = \mathcal{R}(t)\sum_{\tau=1}^{T_g}I(t - \tau)g(\tau)
+
+        \mathcal{R}(t) & = \mathcal{R}^u(t)\exp\left(-\gamma(t)\
+            \sum_{\tau=1}^{T_f}I(t - \tau)f(\tau)\right)
+
+    where :math:`\mathcal{R}(t)` is the reproductive number, :math:`\gamma(t)`
+    is the infection feedback strength, :math:`T_g` is the max-length of the
+    generation interval, :math:`\mathcal{R}^u(t)` is the raw reproduction
+    number, :math:`f(t)` is the infection feedback pmf, and :math:`T_f`
+    is the max-length of the infection feedback pmf.
     """
     feedback_scanner = new_double_scanner(
-        (infection_feedback_pmf, generation_interval_pmf),
-        (jnp.exp, lambda x: x),
+        dists=(infection_feedback_pmf, generation_interval_pmf),
+        transforms=(jnp.exp, lambda x: x),
     )
     latest, infs_and_R = jax.lax.scan(
-        feedback_scanner, I0, (infection_feedback_strength, Rt_raw)
+        f=feedback_scanner,
+        init=I0,
+        xs=(infection_feedback_strength, Rt_raw),
     )
-    return infs_and_R
+
+    infections, rt = infs_and_R
+    rt = rt * Rt_raw
```

Comment:
What is this line doing?

Reply:
As I originally implemented `sample_infections_with_feedback()` and `new_double_scanner` [here](https://github.com/cdcent/cfa-pyrenew/blob/main/src/pyrenew/infection.py), `infs_and_R` should have been a tuple with the infection timeseries and the adjusted $\mathcal{R}(t)$ timeseries, in that order. I just did some basic testing on it and it looks right at first glance, but I can dig deeper.

Reply:
Good catch; that's correct. Probably should rename the variable in that case. Bad on my part initially to call it `infs_and_R` when it's really `infs_and_R_adjustment_factor`

Reply:
Fixed in 1865216